### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by yangshuaipi** [smitheric95/MoCoViT-PyTorch#3]
Hello, is the pre-trained  model ready？

**Comment by smitheric95** [comment]
Hi @yangshuaipi, sorry for the wait. The pre-trained model weights have been uploaded under the `checkpoints` directory.

In full disclosure, I have not been able to reproduce the authors' results on the test set. For some reason, the ImageNet evaluation server returns a 0% accuracy rate despite the output appearing to be correct. I have reached out to their support email, but haven't gotten a response. I'll keep trying to solve this.

The performance of the default checkpoint is 57% Top-1 accuracy on the validation set.
